test/project_types/script/layers/infrastructure/languages/task_runner_test.rb

```diff
@@ -3,17 +3,19 @@
 require "project_types/script/test_helper"
 
 describe Script::Layers::Infrastructure::Languages::TaskRunner do
+  let(:ctx) { TestHelpers::FakeContext.new }
+  let(:language) { "assemblyscript" }
+  let(:script_name) { "script_name" }
+  let(:task_runner) { Script::Layers::Infrastructure::Languages::TaskRunner.for(ctx, language, script_name) }
```

Comment:
I think you just copied and pasted this from previous code, but, the right way to do it is like [this](https://github.com/Shopify/shopify-cli/blob/78377851979e30c037389c20eaf40aa46a406831/test/project_types/script/layers/infrastructure/languages/project_creator_test.rb#L5) in the `project_creator_test.rb`.  Do you mind just fixing it here?  I read forward in POODR, and it's also the suggested way.  Basically, we shouldn't rely on `ASTaskRunner` or any subclasses to exist in order to test `TaskRunner`.